packages/utils/package.json

```diff
@@ -39,7 +39,7 @@
     "@walletconnect/relay-api": "^1.0.9",
     "@walletconnect/safe-json": "^1.0.2",
     "@walletconnect/time": "^1.0.2",
```

Comment:
Perhaps pin all versions to their strict options i.e. remove `^` and re-run npm install without lockfile?